internal/common/constants.go

```diff
@@ -69,6 +69,8 @@ const (
 	EnvGoogleApplicationCredentials = "GOOGLE_APPLICATION_CREDENTIALS"
 	// EnvGoogleStorageAPIEndpoint is the environment variable key for Google storage API endpoint override.
 	EnvGoogleStorageAPIEndpoint = "GOOGLE_STORAGE_API_ENDPOINT"
+	// EnvEmulatorEnabled is the environment variable key for Google storage emulator.
+	EnvEmulatorEnabled = "GOOGLE_EMULATOR_ENABLED"
```

Comment:
This should either be `EnvEmulatorEnabled = "EMULATOR_ENABLED"` or `EnvGoogleEmulatorEnabled = "GOOGLE_EMULATOR_ENABLED"`. Mixing these two creates a conflict with all other providers.

Let's just stick to having the provider as the beginning of the variable since there has been quite a bit of confusion over naming since the beginning of the two PRs here in etcd-druid and their sister PRs in etcd-backup-restore.

If you prefer the other way around then please let me know. I honestly don't have any preference.

```suggestion
	EnvGoogleEmulatorEnabled = "GOOGLE_EMULATOR_ENABLED"
```

Reply:
You're right, I forgot about this. I'll change this. This was not touched after the naming was decided in the etcdbr PR's. `EnvGCSEmulatorEnabled` is the name that is used there, so ig that's better. And for Azure, it's `EnvAzureEmulatorEnabled`

Reply:
Addressed in latest commit [`585d628`](https://github.com/gardener/etcd-druid/pull/741/commits/585d6286183f22ee5700e9bef3b08c2166a3f4ba)

Reply:
The the other two environment variables are of the format `EnvGoogle...`, maybe this could stick to that format too?